exotica/include/exotica/Scene.h

```diff
@@ -230,227 +225,31 @@ namespace exotica
-
-      /**
-       * @brief Get jacobian reference
-       * @param task Task name
-       * @param jac Returned Jacobian reference
-       * @return Indication of success
-       */
-      void getJacobian(const std::string & task, Eigen::MatrixXdRef_ptr& jac,
-          bool force = false);
-
-      /**
-       * \brief	Get joint number N
-       * @return	N
-       */
+      virtual void Update(Eigen::VectorXdRefConst x);
+      void setCollisionScene(const planning_scene::PlanningSceneConstPtr & scene);
+      void setCollisionScene( const moveit_msgs::PlanningSceneConstPtr & scene);
       int getNumJoints();
-
-      /**
-       * \brief	Get end-effector names for a task
-       * @param	task	Task name
-       * @param	effs	Endeffector names
-       * @return Indication of success
-       */
-      void getEndEffectors(const std::string & task,
-          std::vector<std::string> & effs);
-      void getEndEffectors(const std::string & task,
-          std::pair<std::vector<std::string>, std::vector<KDL::Frame>> & effs);
-
-      /**
-       * \brief	Get exotica collision scene ptr
-       * @return	CollisionScene pointer
-       */
       CollisionScene_ptr & getCollisionScene();
-
-      /**
-       * \brief	Get map size for particular taskmap
-       * @param	task	Task name
-       * @return	Map 	Size
-       */
-      int getMapSize(const std::string & task);
-
-      /**
-       * \brief	Get centre of mass properties
-       * @param	seg		Segment names
-       * @param	mass	Mass of each link
-       * @param	cog		Centre of gravity of each link
-       * @param	tip_pose	Tip pose of each link
-       * @param	base_pose	Base pose of each link
-       * @return Indication of success
-       */
-      void getCoMProperties(std::string& task,
-          std::vector<std::string> & segs, Eigen::VectorXd & mass,
-          std::vector<KDL::Vector> & cog, std::vector<KDL::Frame> & tip_pose,
-          std::vector<KDL::Frame> & base_pose);
-
-      /**
-       * \brief	Get task root name
-       * @return	Root name
-       */
       std::string getRootName();
-
-      /**
-       * @brief getPlanningScene Returns the MoveIt! planning scene associated with this Scene
-       * @return Planning Scene
-       */
       planning_scene::PlanningScenePtr getPlanningScene();
-
-      /**
-       * @brief getSolver Returns the instance of Kinematica solver associated with this Scene
-       * @return Kinematic tree solver
-       */
       exotica::KinematicTree & getSolver();
-
-      /**
-       * @brief Returns poses ofrequested end-effectors
-       * @param names List of end-effector names
-       * @param poses Returned poses
-       * @return Indication of success
-       */
-      void getPoses(const std::vector<std::string> & names,
-          std::vector<KDL::Frame> & poses);
-
-      /**
-       * @brief getRobotModel Returns the robot model
-       * @return Robot model
-       */
       robot_model::RobotModelPtr getRobotModel();
-
```

Comment:
I have removed Doxygen where the method/variable name was explicit enough.
The code was becoming hard to edit with all the comments.